src/CSS/ExperimentTwo/ModalStyle.css

```diff
@@ -0,0 +1,31 @@
+.CloseLayout {
+  font-family: sans-serif;
```

Comment:
maybe you want a more specific font?

Reply:
I ended up removing this bit as I realised that I wasn't really using it